lib/swaps/SwapClient.ts

```diff
@@ -330,9 +330,12 @@ abstract class SwapClient extends EventEmitter {
     { remoteIdentifier, units, currency, destination, force, fee }: CloseChannelParams,
   ): Promise<string[]>;
 
-  /** Gets a deposit address. */
+  /** Gets a deposit address for channel. */
```

Comment:
How about "Gets an address for depositing directly to a channel"? I think that'd be a bit clearer in differentiating this from `walletDeposit`